var/spack/repos/builtin/packages/intel-tbb/package.py

```diff
@@ -81,7 +93,8 @@ def coerce_to_spack(self, tbb_build_subdir):
 
     def install(self, spec, prefix):
         if spec.satisfies('%gcc@6.1:') and spec.satisfies('@:4.4.3'):
-            raise InstallError('Only TBB 4.4.4 and above build with GCC 6.1!')
+            raise \
+                InstallError('Only TBB 4.4.4 and above build with GCC >= 6.1!')
```

Comment:
Can you actually convert this to a `conflicts` statement? That will cause Spack to crash during concretization instead of installation. It would look something like:
```python
conflicts('%gcc@6.1:', when='@:4.4.3', msg='Only TBB 4.4.4 and above build with GCC >= 6.1!')
```